File: src/py-dev/LiENa/LiENaStructure/LiENaMessage/LienaDisengagementCommitMessage.py
from LiENaStructure.LiENaMessage.LienaMessage import LienaMessage


class LienaDisengagementCommitMessage(LienaMessage):
    def __init__(self, message_id, target_id, origin_id, timestamps, dlc):
        LienaMessage.__init__(self, message_id, target_id, origin_id, timestamps, dlc)
        self.count = 0

File: src/cxx-dev/LiENa/lienaStructure/lienaMessage/lienaDisengagementCommitMessage.cpp
#include "lienaDisengagementCommitMessage.h"


/**
 * @brief lienaDisengagementCommitMessage::lienaDisengagementCommitMessage
 * @param messageId
 * @param targetId
 * @param timestampes
 * @param DLC
 */
lienaDisengagementCommitMessage::lienaDisengagementCommitMessage(uint64_t messageId,
                                                                 uint32_t targetId,
                                                                 uint32_t originId,
                                                                 uint64_t timestampes,
                                                                 uint32_t DLC) : lienaMessage(messageId, targetId, originId,timestampes, DLC)
{


}

//! ----------------------------------------------------------------------------------------------
//!
//! \brief lienaDisengagementCommitMessage::~lienaDisengagementCommitMessage
//!
lienaDisengagementCommitMessage::~lienaDisengagementCommitMessage(){

}
